orttraining/orttraining/python/training_session.py

```diff
@@ -0,0 +1,42 @@
+#-------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+#--------------------------------------------------------------------------
+
+import sys
+import os
+
+from onnxruntime.capi import _pybind_state as C
+from onnxruntime.capi import InferenceSession, IOBinding
+
+
+class TrainingSession(InferenceSession):
```

Comment:
>TrainingSession [](start = 6, length = 15)

This is copied from Session.py right?

Do we need to clean up from Session.py ?